Library/Formula/gettext.rb

```diff
@@ -9,10 +9,11 @@ class Gettext < Formula
     sha256 "c64bb31029e29599442653fe1e0f2216ae8fe144451a0541896f6e367df0018f" => :tiger_altivec
   end
 
-  keg_only :shadowed_by_osx, "OS X provides the BSD gettext library and some software gets confused if both are in the library path."
+  unless MacOS.version <= :leopard  # what Mac OS version would be correct here?
```

Comment:
Huh, does Tiger *not* include gettext? I don't think I knew that!

Reply:
Tiger does not include gettext, nor any fragment thereof.  Neither does Leopard.  I have no idea when it became a standard component, nor indeed whether it ever actually did.

Reply:
I posted a bug report to that effect months ago.